ui/app/src/services/content.ts

```diff
@@ -807,23 +812,14 @@ export function getChildrenByPath(
   );
 }
 
-export function fetchItemsByPath(
-  site: string,
-  paths: string[],
-  options?: {
-    skipHomePathOverride?: boolean;
-  }
-): Observable<DetailedItem[]> {
-  const requests: Observable<DetailedItem>[] = [];
-  paths.forEach((path) =>
-    requests.push(
-      getDetailedItem(
-        site,
-        path === '/site/website' ? (options?.skipHomePathOverride ? path : '/site/website/index.xml') : path
-      )
+export function fetchItemsByPath(siteId: string, paths: string[], preferContent = true): Observable<DetailedItem[]> {
```

Comment:
LLeave the third arg as an object of options. Default it to true. Create an interface just like with get children.